bundles/io/org.eclipse.smarthome.io.rest.test/src/test/groovy/org/eclipse/smarthome/io/rest/test/filter/CorsFilterTest.groovy

```diff
@@ -34,6 +34,8 @@ class CorsFilterTest {
     private static final String HTTP_GET_METHOD = "GET"
     private static final String HTTP_OPTIONS_METHOD = "OPTIONS"
 
+    private static final String CONTENT_TYPE_HEADER = "content-type";
```

Comment:
Should not just the `containsHeaderWithValue` be modified to check all values like case-insensitive? Because all header names are case-insensitive and therefore having a separate method `containsHeaderWithCaseInsensitiveValue` to check case-insensitive values will make the current method of no use. No?